lib/email/sender.rb

```diff
@@ -354,43 +354,45 @@ def digest_posts
       Post.where(id: header_value("X-Discourse-Post-Ids")&.split(","))
     end
 
-    def add_attachments(post)
+    def add_attachments(*posts)
```

Comment:
Moving this here means we don't need to significantly change the implementation of `#add_attachments`. The e-mail size check still works, and `#fix_parts_after_attachments!` is called just once.